libuuu/fastboot.cpp

```diff
@@ -266,8 +269,11 @@ int FBUpload::run(CmdCtx* ctx)
 	cmd.format("upload");
 
 	std::vector<uint8_t> buff;
-	if (fb.Transport(cmd, nullptr, buff.size(), &buff))
+	if (fb.Transport(cmd, nullptr, buff.size(), &buff)) {
+		string err = "Error on transport !";
+		set_last_err_string(err);
```

Comment:
All error code setting at leaf node.  For here,  "error on transport" should be set in fb.transport() function, where actuall return -1.

Reply:
Done.